streams/jvm/src/main/scala/zio/stream/platform.scala

```diff
@@ -236,4 +239,84 @@ trait ZStreamPlatformSpecificConstructors { self: ZStream.type =>
    */
   final def fromJavaStreamTotal[A](stream: => ju.stream.Stream[A]): ZStream[Any, Nothing, A] =
     ZStream.fromJavaIteratorTotal(stream.iterator())
+
+  /**
+   * Create a stream of accepted connection from server socket
+   * Emit socket `Connection` from which you can read / write and ensure it is closed after it is used
+   */
+  def fromSocketServer(
+    port: Int,
+    host: Option[String] = None
+  ): ZStream[Blocking, Throwable, UManaged[Connection]] =
```

Comment:
What do you think about using ZManaged#scope to pull the manageds into the stream itself?
This leads to a far more ergonomic signature and is still resource safe as all connections will be closed when the stream is closed.

Somethinghttps://github.com/zio/zio-keeper/blob/master/keeper/src/main/scala/zio/keeper/transport/tcp.scala like this

Reply:
Yes I think that'd be good. `ZStream[Connection]` is indeed nicer than `ZStream[Managed[Connection]]`. What do you think @regis-leray?

Reply:
👍